platform/android/app/build.gradle

```diff
@@ -95,13 +98,16 @@ android {
             }
         }
     }
-
     packagingOptions {
-        exclude 'META-INF/DEPENDENCIES'
+        resources {
+            excludes += ['META-INF/DEPENDENCIES']
+        }
     }
+
 }
 
 repositories {
+    // used for carouselView
```

Comment:
Right. Should maybe have been documented, but it should be in the commit message.
I suggested in #580 to get rid of it. Maybe it's about time.

Reply:
The repository we can't remove yet, because the CarouselView is used for the onboarding, AFAIK.

Reply:
No, I mean CarouselView. There are newer, and updated, Kotlin alternatives. This is just an old Java library.

Reply:
Got it!